### ISSUE TITLE
No predicted in graph

### ISSUE THREAD

**Issue by BNGdev** [yacoubb/stock-trading-ml#9]
Thanks a lot, Yacoub. It's an interesting project.

I want to make it running on Jupyter Notebook but can't make it work yet. I'm new here, still trying...

Under Anaconda, I run your code, and found 2 issues:

1/ set_random_seed() can't be imported, I made a minor update to fix it.
#from tensorflow import set_random_seed
#set_random_seed(4)
import tensorflow
tensorflow.random.set_seed(4)

2/ I run my data but the final graph could not show the predicted line. Here are some of the outputs. Does it mean something wrong with my data?
Using TensorFlow backend.
(4535, 50, 5)
(504, 50, 5)
2020-05-17 00:47:18.540599: I tensorflow/core/platform/cpu_feature_guard.cc:142] Your CPU supports instructions that this TensorFlow binary was not compiled to use: AVX AVX2
Train on 4081 samples, validate on 454 samples
Epoch 1/50
4081/4081 [==============================] - 3s 626us/step - loss: nan - val_loss: nan
Epoch 2/50
4081/4081 [==============================] - 2s 537us/step - loss: nan - val_loss: nan
Epoch 3/50

**Comment by XyzzyX44** [comment]
Check out this post. It basically states that the issue is with your CPU. If you have a supported GPU card, you could set your system up to work with it. 

https://stackoverflow.com/questions/47068709/your-cpu-supports-instructions-that-this-tensorflow-binary-was-not-compiled-to-u
